cnv/cnv-release-notes/cnv-release-notes.adoc

```diff
@@ -42,8 +42,10 @@ This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=171
 
 == Known issues
 
-* If you have {CNVProductName} 2.1.0 deployed, you must first upgrade to 2.2.0
-before upgrading {product-title}.
+* If you have {CNVProductName} 2.1.0 deployed, you must first upgrade {CNVProductName}
+to 2.2.0 before upgrading {product-title}. Upgrading {product-title} before upgrading
+{CNVProductName} might trigger virtual machine deletion.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1785661[*BZ#1785661*]).
```

Comment:
super tiny nit: take out the period after the BZ link to match the others.

Reply:
Removed.